postgresql/db-common-archiver-status.yml

```diff
@@ -0,0 +1,28 @@
+type: db
+data:
+  query: SELECT * FROM pg_stat_archiver;
+  dbUrl: jdbc:postgresql://${POSTGRESQL_DB_HOST_PORT}/${POSTGRESQL_DB_DATABASE}
+  dbDriverClass: org.postgresql.Driver
+  dbUser: ${POSTGRESQL_DB_USER}
+  dbPassword: ${POSTGRESQL_DB_PASSWORD}
+  dbAdditionalConnectionParams: ${POSTGRESQL_DB_ADDITIONAL_PARAMS}
+  #  dbVerticalModel="true" - because output of SHOW STATUS command gives each attribute in its own row
```

Comment:
This comment should be removed. It is present in other files, remove from there too please.

Reply:
Removed this comment on all postgresql yml files.